mirrord-agent/src/sniffer.rs

```diff
@@ -236,13 +237,29 @@ pub async fn packet_worker(
     mut rx: Receiver<SnifferCommand>,
     interface: String,
     container_id: Option<String>,
+    container_runtime: Option<String>,
 ) -> Result<()> {
     debug!("setting namespace");
-    if let Some(container_id) = container_id {
-        let namespace = get_container_namespace(container_id).await?;
-        debug!("Found namespace to attach to {:?}", &namespace);
-        set_namespace(&namespace)?;
+
+    let default_runtime = "containerd";
```

Comment:
Same thing as my other comment, if this never changes, move it outside as a `const`.